auth/src/main/java/org/openedx/auth/presentation/signin/SignInFragment.kt

```diff
@@ -125,6 +106,7 @@ class SignInFragment : Fragment() {
 internal sealed interface AuthEvent {
     data class SignIn(val login: String, val password: String) : AuthEvent
     data class SocialSignIn(val authType: AuthType) : AuthEvent
+    data class OpenLink(val links: Map<String, String>, val link: String) : AuthEvent
```

Comment:
we can pass the title and URL here.
```
data class OpenLink(val title: String, val link: String) : AuthEvent

Reply:
all my logic is executed in the view model level